### DISCUSSION TITLE
values and values files , differences?

### DISCUSSION THREAD

**Discussion by 1mfaasj** [argoproj/argo-cd#8050] [Q&A]
Hi
Iam trying to create a new Helm application from a Helm repository. Ive got 2 options here: Values files and Values.
What is the difference?
![image](https://user-images.githubusercontent.com/71582854/147604478-f1abad38-1d54-4a5f-aa57-de6100c3270d.png)

**Comment by pasha-codefresh** [comment]
Values files it is YAML files that are located inside your helm chart , In case with values it is just key=value

See here 

https://argo-cd.readthedocs.io/en/stable/user-guide/helm/#values-files
